host/class/uvc/usb_host_uvc/include/usb/uvc_host.h

```diff
@@ -14,13 +14,30 @@
 // Use this macros for opening a UVC stream with any VID or PID
 #define UVC_HOST_ANY_VID (0)
 #define UVC_HOST_ANY_PID (0)
+#define UVC_HOST_ANY_DEV_ADDR (0)
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 typedef struct uvc_host_stream_s *uvc_host_stream_hdl_t;
 
+enum uvc_host_driver_event {
+    UVC_HOST_DRIVER_EVENT_DEVICE_CONNECTED = 0x0,
+};
+
+typedef struct {
+    enum uvc_host_driver_event type;
+    union {
+        struct {
+            uint8_t dev_addr;
+            uint8_t iface_num;      //!< Disconnection event
```

Comment:
I'm not sure why we need to pass `iface_num` to the user... Do you have any example of usage?

Reply:
I misunderstood the meaning of uvc_host_stream_config_t: uvc_stream_index. We should change iface_num to uvc_stream_index here.

If a camera supports multiple video streams, the callback will be called multiple times, with the uvc_stream_index being returned incrementally each time.